core/src/main/groovy/noe/ews/server/httpd/HttpdRpm.groovy

```diff
@@ -63,13 +63,13 @@ class HttpdRpm extends Httpd {
     return new File("/var/run/${serviceName}/${serviceName}.pid")
   }
 
-  Integer extractPid() {
+  Long extractPid() {
     def pidAsStr = new ByteArrayOutputStream()
     if (Cmd.executeCommandRedirectIO("sudo cat ${getPidFile()}", null, null, pidAsStr, System.err) == 0) {
       pidAsStr = pidAsStr.toString()
       pidAsStr = pidAsStr.trim()
       pidAsStr = pidAsStr.replaceAll('"', '')
-      pid = Integer.valueOf(pidAsStr)
+      pid = Long.parseLong(pidAsStr)
```

Comment:
Any reason why you are using Long.parseLong() instead of Long.valueOf() ?

Reply:
hm, truth, I suppose that we should stick with what it was here before. Also, since we are supposed to return `Long` (function definition), we actually should rather use `valueOf`. Updated.

Reply:
@AdamKrajcik, I've updated also other occurrences of the `parseLong` back to `valueOf` where applicable.